index.html

```diff
@@ -1115,25 +1115,60 @@ <h4>Listing</h4>
 
                     <section id="validation" class="normative">
                         <h4>Validation</h4>
-                        <span class="rfc2119-assertion" id="td-validation-syntactic">
-                            The syntactic validation of TD objects before storage is RECOMMENDED to
-                            prevent common erroneous submissions.
-                        </span>
-                        <span class="rfc2119-assertion" id="td-validation-jsonschema">
-                            The server MAY use
-                            <a href="https://www.w3.org/TR/wot-thing-description/#json-schema-for-validation">Thing Description JSON Schema</a>
-                            to validate standard TD vocabulary, or a more comprehensive JSON Schema to also validate extensions.
-
-                        </span>
-                        <span class="rfc2119-assertion" id="td-validation-result">
-                            If the server fails to validate the TD object, it MUST inform the client
-                            with necessary details to identify and resolve the errors.
-                        </span>
-
-                        <p class="ednote" title="Validation Example">
-                            It would be nice to include an example validation result in
-                            the uniform error format (i.e. RFC7807 Problem Details).
+                        <p>
+                           <span class="rfc2119-assertion" id="td-validation-syntactic">
+                                The syntactic validation of TD objects before storage is RECOMMENDED to
+                                prevent common erroneous submissions.
+                            </span>
+                            <span class="rfc2119-assertion" id="td-validation-jsonschema">
+                                The server MAY use
+                                <a href="https://www.w3.org/TR/wot-thing-description/#json-schema-for-validation">Thing Description JSON Schema</a>
+                                to validate standard TD vocabulary, or a more comprehensive JSON Schema to also validate extensions.
+                            </span>
                         </p>
+                        <p>
+                            <span class="rfc2119-assertion" id="td-validation-result">
+                                If the server fails to validate the TD object, it MUST inform the client
+                                with necessary details to identify and resolve the errors.
+                            </span>
+    
+                            <span class="rfc2119-assertion" id="td-validation-response">
+                                The validation error SHOULD be described as Problem Details [[RFC7807]]
+                                and extended with a `validationErrors` array of objects
```

Comment:
... with an extension...; is there a data schema for this in the TD?

Reply:
Not yet. It depends in being able to define an schema for additionalResponses: https://github.com/w3c/wot-thing-description/issues/1053

Reply:
We basically agreed that we would support this in TDs although I need to finalize the PR for it so it can be merged.  Rather than putting the data schema in the form directly they will be in a named map like securityDefinitions, though.